### DISCUSSION TITLE
Simple browser automation example

### DISCUSSION THREAD

**Discussion by bhougland18** [keathley/wallaby#723] [Q&A] [answered]
Can Wallaby be used as a tool for simple browser automation similar to the following?

https://github.com/HashNuke/hound/blob/master/notes/simple-browser-automation.md

Thank you!

**Comment by mhanberg** [comment]
Yes.

**Reply by bhougland18** [reply]
I guess I am stuck because it seems most of the documentation is around teasting you own app.  What would Wallaby's version of the Hound snippet look like? Would the project still be structured the same way?

**Comment by mhanberg** [comment, marked as answer]
```elixir
Mix.install([:wallaby])

Application.ensure_all_started(:wallaby)

defmodule Example do
  use Wallaby.DSL

  def run do
    {:ok, session} = Wallaby.start_session()

    session
    |> visit("http://akash.im")
    |> page_title()
    |> IO.inspect()
  
   Wallaby.end_session(session)
  end
end

Example.run()
```
